src/ProjectTemplates/BlazorTemplates.Tests/BlazorTemplates.Tests.csproj

```diff
@@ -68,6 +68,10 @@
       <_Parameter2>$(TargetFramework)</_Parameter2>
       <_Parameter3></_Parameter3>
     </AssemblyAttribute>
+    <AssemblyAttribute Include="System.Reflection.AssemblyMetadataAttribute">
+      <_Parameter1>Test.RazorSdkDirectoryRootOverride</_Parameter1>
+      <_Parameter2>$(RazorSdkDirectoryRoot)</_Parameter2>
```

Comment:
The more I see of the `RazorSdkDirectoryRootOverride` name, the less I think it's needed